sdk/appcenter/src/main/java/com/microsoft/appcenter/persistence/Persistence.java

```diff
@@ -32,7 +32,7 @@
      * @param log   The log to be placed in the storage.
      * @throws PersistenceException Exception will be thrown if Persistence cannot write a log to the storage.
```

Comment:
What is the return value? can you add a returns tag?

Reply:
Needed for new unit tests. Will add Javadoc comment.